ipmininet/install/build_vm.sh

```diff
@@ -30,8 +33,9 @@ sudo pip3 install --upgrade pip
 sudo apt remove -yq python3-pip
 
 # Install ipmininet
-git clone https://github.com/cnp3/ipmininet.git
+[ ! -d "ipmininet" ] && git clone ${IPMN_REPO}
```

Comment:
This is a great idea, could take this one step further and add a variable for the clone target?
E.g.
```suggestion
IPMN_DIR="${IPMN_DIR:-ipmininet}"
[ ! -d "${IPMN_DIR" ] && git clone ${IPMN_REPO}${IPMN_DIR}
pushd ${IPMN_DIR}
```

Reply:
> What's preventing us from directly following the steps from the OpenR repo?
I.e., https://github.com/facebook/openr/tree/master/build#build-on-debian-host-without-docker

Main problem is that the generated build scripts do not pin dependencies properly. My idea was to adjust the generated script and pin dependencies to create a stable build script for the 'ipmininet supported' release.

Reply:
:+1: for the change

Reply:
See 6fd96f1 (I added a missing `}` and ` ` to your suggestion)